A proposed version of v0.44

senor-huevo opened the pull request:
Basically  couple of balance of changes, Adds text to certain cards explaining their past and present, and rewrites a few sigils to make them more consistent.

Undead Cat
1-Blood --> 3-Blood
Gains Unkillable 

Cat is in a strange position; it has nothing interesting about it, So why not make it a sort of win-con whence you release Undead Cat.

Mole
0/4 --> 0/5

Its so ass,  but there's not much to be done other than give it a slight health buff.

Strange Pupa
1-Blood --> 2-blood
0/3 --> 0/4

actually makes it so much more threatening without raising past any damage thresholds.

Mothman
1-Blood --> 3-Blood
7/3 --> 7/5

this raises it past multiple damage thresholds, but that's after you left it live for 2 turns

Squirrel Ball
0/1 --> 0/2

Gives it the chance to the chance to see some play instead of being useless forever

Gamblobot
3-Energy --> 4-Energy
Remove Energy Dice (2)
Gains Energy Dice

same effective cost, but now you have an extra chance to gamble if you want to go all in.

Insectodrone
3-Energy --> 2-Energy
1/2 --> 1/1

3-Energy for a 1/2 with airborne is technically on curve but pales in comparison to S0N1A

Rustbucket
2/2 --> 2/3

it's worse than the alternatives because of its sigil, but at least now it has the stats to be on par with them

Bonehound
7-Bones --> 6-Bones

It's so fucking ass at 7-Bones, it never sees play; it's some how more expensive than Bolthound

Mummy Lord
8-Bones --> 7-Bones
3/3 --> 3/2

Turkey Vulture is just so much better than it, so why not make it on par with a wolf

Sarcophagus
4-Bones --> 3-Bones
gains Bloodless

never sees play, but would also would need to be cheaper since Mummy Lord is cheaper

Voodoo Doll
3-Bones --> Free
Gains Bloodless
never sees play since paying 6-Bones just to activate its ability once is so terrible

Mox tower
0/2 --> 0/3
Gains Bloodless

it never made sense that it provided blood

Mox Bunker
0/2 --> 0/3
Gains Bloodless

it never made sense that it provided blood

Mox Encampment
0/2 --> 0/3
Gains Bloodless

it never made sense that it provided blood

Dragon Egg
Remove Fledgling and Gem Dependent
Gains Fledgling 2

Pointless as is, fair since it's like a mini Mothman. Also to make sure it's not broken with Holo Mox.

Ruby Dragon
2/2 --> 2/3
Remove Gem Dependent
Gains Airborne

This is so it's a proper win-con

Holo Mox
2-Energy and Orange Gem --> 2-Energy
Gains Orange Mox
Gains Bloodless

Literally had no reason to exist, now it's actually a card to be used.

Winged Guardian
1/3 --> 1/4

it was so ass for so long, it at least deserves something 

Leaping Vessel
2-Energy --> 1-Energy

So ass

Splintered Vessel
2-Energy --> 1-Energy

So ass

Contaminated Vessel
2-energy --> 1-Energy

So Ass


Acid Squirrel
Common --> Rare

Consistency

Now Vessels are a draft instead of being predrafted.

senor-huevo committed:
My Proposed changes to IMF Standard.

senor-huevo committed:
Add files via upload

107zxz commented:
Big fan of all of these changes. I also really appreciate the text and wording changes to sigil descriptions! Thank you so so much. I approve of all of this :)

senor-huevo committed:
Making the finishing touches

107zxz reviewed (approved):
Re approving

senor-huevo committed:
Sorting, special thanks to Mazzor

Mazzor helped sort this; also Obol 1 now costs 2-Bones

senor-huevo commented:
The Actual Finishing touches this time. Got help from Mazzor for Sorting this; Broken Obol costs 2-bones now because it looked weird not being beside Broken Obol-2. Now just waiting for the next update.

senor-huevo commented:
the reason I left it as a draft besides waiting for the next update is because that there are some things that I'm iffy on new bools that I'm adding such as `vanilla_gem_dep` and `lesser`.

senor-huevo committed:
Updated Proposed version of 0.44

senor-huevo committed:
small tweak

senor-huevo commented:
Updated proposed 0.44

- Geck
Unbanned

- Undead Cat
1-Blood -> 2-Blood

- Bell Tentacle
 Common -> Rare

- Hand Tentacle
Common -> Rare

- Mirror Tentacle
Common -> Rare

- Mole
+Bone Digger


- Raven Egg
0/2 -> 0/3

- Acid Squirrel
Common -> Rare

- Squirrel Ball
0/1 -> 0/2

- Strange Pupa
0/3 -> 0/4

- Mothman
7/3 -> 7/5

- Null Conduit
1-energy -> Free

- Insectodrone
3-energy -> 2-Energy
1/2 -> 1/1

- Winged Guardian
1/3 -> 1/4

- Leaping Vessel
2-Energy -> 1-Energy

- Splintered Vessel
2-Energy -> 1-Energy

- Contaminated Vessel
2-Energy -> 1-Energy

- Alarmbot
3-Energy -> 2-Energy

- Factory Conduit
3-Energy -> 2-Energy

- GR1ZZ
0/4 -> 0/6

- GR1ZZ (beast)
4/4 -> 4/6

- Rustbucket
4-Energy -> 3-Energy

- Steel Mice
- 4-Energy -> 3-Energy

- QU177
5-Energy -> 4-Energy

- QU177 (beast)
5-Energy -> 4-Energy

- Bolthound
6-Energy -> 5-Energy

- Steambot
6-Energy -> 5-Energy

- Voodoo Doll
3-Bones -> 1-Bone

- Ghost Ship
+ Bloodless

- Sarcophagus
4-Bones -> 3-Bones
+ Bloodless

- Drowned Soul
4-Bones -> 3-Bones

- Mummy Lord
8-Bones -> 7-Bones

- Dragon Egg
- Fledgling, Gem Dependent
+ Fledgling 2

- Ruby Dragon
2/2 -> 2/3
Airborne

- Holo Mox
- Orange Gem
+ Orange Mox
+ Bloodless

- Mox Encampment
0/2 -> 0/3
+ Bloodless

- Mox Tower
0/2 -> 0/3
+ Bloodless
- 
- Mox Bunker
0/2 -> 0/3
+ Bloodless

- Moon Shard
6-Energy -> 4-Energy

-Removed 3 Gecks sidedeck

- Merged Vessels into a draft

Didn't do the sigil rewrites because I'm still working those out

senor-huevo commented:
Also credited syntax's art that went uncredited
